### ISSUE TITLE
Disable analytics option has wrong behavior

### ISSUE THREAD

**Issue by nicolasvienot** [meilisearch/MeiliSearch#1983]
**Describe the bug**

Following the [documentation](https://docs.meilisearch.com/reference/features/configuration.html#disable-analytics), we should be able to disable/enable the built-in telemetry by setting `--no-analytics` or `MEILI_NO_ANALYTICS` to `true`/`false`.

This is not the current behavior of `MeiliSearch`, which is :
- If the variable is set (to anything), `MeiliSearch` built-in telemetry is disabled
- If the variable is not used, `MeiliSearch` built-in telemetry is enabled

**To Reproduce**

```bash
~ MEILI_NO_ANALYTICS=true ./meilisearch-0.24.0 -> Analytics OFF
~ MEILI_NO_ANALYTICS=false ./meilisearch-0.24.0 -> Analytics OFF
~ MEILI_NO_ANALYTICS="" ./meilisearch-0.24.0 -> Analytics OFF
~ ./meilisearch-0.24.0 --no-analytics true -> Analytics OFF
~ ./meilisearch-0.24.0 --no-analytics false -> Analytics OFF
~ ./meilisearch-0.24.0 --no-analytics "" -> Analytics OFF

~ ./meilisearch-0.24.0 -> Analytics ON
```

**Expected behavior**
```bash
~ MEILI_NO_ANALYTICS=false ./meilisearch-0.24.0 -> Analytics ON
~ ./meilisearch-0.24.0 --no-analytics false -> Analytics ON
```

OR Documentation update. @meilisearch/docs-team 

**MeiliSearch version:** 

`v0.24.0`, `v0.23.1`

**Comment by curquiza** [comment]
Thanks for the detailed issue @nicolasvienot

**Comment by curquiza** [comment]
Closed by #1984
